CHANGES.rst

```diff
@@ -62,6 +62,10 @@ New Features
   - The ``spectral_density`` equivalency now supports transformations of
     luminosity density. [#5151]
 
+  - ``Quantity`` now accepts ``Unit`` objects and strings that can be parsed by
+    the ``Unit`` constructor, i.e., ones consisting of a number and unit such
+    as '10 km/s' [#1773].
```

Comment:
@adrn I’d argue it *contains* a valid unit, but isn’t one. It’s like asking “What’s the currency of England?” and getting “10 £” as the reply. Pounds is correct, but "10 £" is a specific value (`Quantity`). (Though maybe `Unit` is already parsing things like that; I don't know.)